lib/KBaseReportPy/utils/report_utils.py

```
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+from file_utils import validate_paths, fetch_or_upload_files
+from uuid import uuid4
+
+""" Utilities for creating reports using DataFileUtil """
+
+
+def create_report(ctx, params, dfu):
+    """
+    Create a simple report
+    :param ctx: context dict passed into KBaseReportImpl#create
+    :param params: see utils/validation_utils
+    :param dfu: instance of DataFileUtil
+    :returns: report data
+    """
+    report_name = "report_" + str(uuid4())
```

Seems like this should throw an error if unsupported fields are included (*_links for instance)